frontend/src/components/primitives/Textarea/Textarea.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { TextArea as TextAreaComponent, TextAreaProps } from './Textarea';
+import { TextArea as TextAreaComponent, TextAreaProps } from './TextArea';
```

Comment:
@JakubVavra toto nie je chyba ? V projekte to vidim s malym a, teda Text**a**rea, nie s velkym Text**A**rea. Zcheckni prosim este

Reply:
Opraveno